packages/shared/src/features/onboarding/steps/FunnelOrganicSignup.tsx

```diff
@@ -102,13 +123,30 @@ export const FunnelOrganicSignup = withIsActiveGuard(
     );
 
     useEffect(() => {
-      if (isLoggedIn && isAuthReady && !!user.infoConfirmed) {
+      if (!isAuthReady) {
+        return;
+      }
+
+      if (
+        isLoggedIn &&
+        !!user.infoConfirmed &&
+        !hasAlreadyCheckedUser.current
+      ) {
```

Comment:
We need to send user to next step only if is confirmed on first load. 
When social registration occurs you have a confirmed user that still needs to complete the profile. Add this check avoid the transition on those cases.